src/site/content/en/blog/earth-webassembly/index.md

```diff
@@ -0,0 +1,67 @@
+
+Other browsers are working on Site Isolation or other mitigations in order to re-enable SharedArrayBuffer. In the meantime, Earth runs single-threaded in those browsers.
+
+<img class="w-screenshot" src="GoogleEarthEiffel.jpg" alt="A screenshot of Earth showing Eiffel Tower">
+
+
+## How WebAssembly works with different browsers
+
+We’ve learned a lot about the state of WebAssembly support in browsers porting Earth. If you’re going to develop applications using WebAssembly, it’s important to understand the current state of how WebAssembly works with different browsers.  
+
+### Edge: 
+Edge is on the verge of becoming two distinct development experiences based on Microsoft's choice to move from the EdgeHTML renderer over to a Chromium-based renderer moving forward. At the moment, the Google Earth beta on WebAssembly won’t run on the current public version of Edge due to lack of support for WebGL2—but that’ll be fixed once the new version of Edge, based on Chromium, ships in the near future. In the meantime, you can [download the Canary version of Edge](https://www.microsoftedgeinsider.com/download) and see that Earth works quite well.
+
+### Chrome:
+Chrome has strong support for WebAssembly, including multi-threading on desktop, so you can expect Earth to run smoother as a result. However, we look forward to Chrome adding support for dynamic memory allocation with multi-threading in WebAssembly. Until then, Earth may fail to start on devices with limited amounts of memory (such as 32-bit machines).
+
+### Firefox: 
+Firefox offers good support for WebAssembly, but has disabled support for multi-threading. As a result, you can expect a slower experience with Earth. We look forward to Mozilla bringing back support for multi-threading in future versions. On the upside, Firefox does support dynamic memory allocation.
+
+### Opera: 
+Opera is based on Chromium just as Chrome is, along with upcoming versions of Edge. However, the current version of Opera only offers single-threaded support of WebAssembly. Earth does run in Opera, but at a somewhat degraded experience. Hopefully newer versions of Opera will have support for multi-threading and more robust WebAssembly support.
+
+### Safari: 
+Safari has a strong implementation of WebAssembly, but it lacks full support for WebGL2. Therefore, Earth with WebAssembly does not run in Safari. Specifically, some of our shaders require GLSL 1.2. We hope that Earth will be available on Safari as well, once better support for WebGL2 is added.
+
+## Looking forward to more adoption of WebAssembly features
+It’s been a long road to make Earth available on the web at its launch, and today with our beta, a first step toward better cross-browser experiences. About six years ago, we started with an initial [asm.js](http://asmjs.org/)-based internal demo that was maintained and expanded over the years. It was then converted into a WebAssembly build of Earth, as WebAssembly became the W3C adopted standard. 
```

Comment:
Suggestion to lean the first sentence a bit: "It's been a long road to make Earth available on the web." I'm suggestion to remove 'at its launch, and today with our beta, a first step toward better cross-browser experiences'. That's been covered.

Reply:
fixed